src/add-liquidity/v2/common.ts

```diff
@@ -39,15 +39,14 @@ export async function execute({
     const [{confirmedRound, txnID}] = await sendAndWaitRawTransaction(client, [
       signedTxns
     ]);
-    const assetOutInnerTxn = (await getAppCallInnerTxns(client, txGroup))?.find(
-      (item) => item.txn.txn.type === "axfer"
-    )?.txn.txn;
+    const assetOut = (await getAppCallInnerAssetData(client, txGroup))?.find(
+      // Output asset is the pool token for add liquidity
+      ({id}) => id === pool.poolTokenID
+    );
```

Comment:
Should the execute raise an error if getAppCallInnerAssetData errors out for some reason? I don't think we would want it to look like the add liquidity operation (or other operations where we use getAppCallInnerAssetData) was failed if the error was related to us trying to read assetOut amount.

Reply:
you're right, fixed it